.env_example

```
@@ -12,6 +12,13 @@ AWS_SECRET_ACCESS_KEY='minioadmin'
 
 #### Airflow connections
 FILE_CONN_ID="MINIO_LOCAL" 
-AIRFLOW_CONN_MINIO_LOCAL='{"conn_type": "aws", "login": "minioadmin", "password": "minioadmin", "extra": "{\"aws_access_key\": \"minioadmin\", \"aws_secret_access_key\": \"minioadmin\", \"endpoint_url\": \"http://host.docker.internal:9000\"}"}'
+AIRFLOW_CONN_MINIO_LOCAL='{
+    "conn_type": "aws", 
+    "extra": {
+        "aws_access_key_id": "minioadmin", 
+        "aws_secret_access_key": "minioadmin", 
+        "endpoint_url": "http://host.docker.internal:9000"
+        }
+    }'
 AIRFLOW_CONN_AWS_DEFAULT=${AIRFLOW_CONN_MINIO_LOCAL}
-AIRFLOW_CONN_MLFLOW_DEFAULT='{"conn_type": "http", "host": "http://host.docker.internal", "port": 5000, "login": ""}'
\ No newline at end of file
+AIRFLOW_CONN_MLFLOW_DEFAULT='{"conn_type": "http", "host": "http://host.docker.internal", "port": 5000, "login": ""}'
```

```suggestion
AIRFLOW_CONN_MLFLOW_DEFAULT='{"conn_type": "http", "host": "http://host.docker.internal", "port": 5000, "login": "admin", "password": "admin"}'
```

This fixes an issue arising by a recent change in the MLflow provider.